<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">

<html lang="en-GB">

<head>
  <title>Micron Oxford Advanced Bioimaging: News and events page</title>
  <meta name=description content="Micron Oxford News and Media">
  <meta name=keywords content="Micron Oxford news events media outreach gallery Wellcome Trust">
  <meta http-equiv="Content-Type" CONTENT="text/html; charset=UTF-8">
  <link rel="stylesheet" type="text/css" href="styles/main.css" >
  <link rel="stylesheet" type="text/css" href="styles/news.css" >
  <link rel="icon" type="image/ico" href="images/MicronMu.ico" >
</head>

<!-- Body has container_all which is divided into header, navigation, mainbody, footer -->
<body id="news">
  <div id="container_all">

    <?php include "headnav.html"; ?>

    <div id="mainbody">

      <div id="leftpanel">

        <h3>Outreach activities</h3>

        <p>
          We are committed to communicating our work to
          both the scientific community and the lay audience. For a Plain
          English description of our activities, please read more
          <a href="about#plain_english"> About Us.</a>
        </p>
        
             <h4>Wellcome Image Awards 2017</h4>
        <p>Congratulations to Ezequiel Miron from the <a href="https://www.bioch.ox.ac.uk/aspsite/index.asp?pageid=882">Schermelleh Group</a>, Department of Biochemistry at the University of Oxford who's image is one of the selected winners of the Wellcome Image Awards 2017.</p>
<p>His picture, captured using the <a href="http://www.micron.ox.ac.uk/facilities/OMXV3">OMX V3 super-resolution microscope</a> in Micron, shows the nucleus of one of two new daughter cells. The DNA in this cell has somehow become caught, and is being pulled between the two dividing cells causing the DNA to be stretched out inside the nucleus; DNA fibres can be seen running through it. As the new cells have moved apart the DNA is being pulled through a perforation in the nuclear membrane. The tension distributed by the rope-like DNA has deformed the usually circular envelope of the nucleus.
        </p>
        <img src="images/ezequiel.png" width="560" align="right" />
        <h4>
          Unravelled DNA in a human lung cell
        </h4>

         <p>&nbsp;</p>
          <p>&nbsp;</p>
        <h4>Outreach Activities: <a href="http://www.oxfordshiresciencefestival.com/">Oxford Science Festival 2016</a></h4>
        
        <p>On the 25th and 26th of June, Micron was involved in setting up one of thirty stalls at the Oxford Science Fair, part of the Oxford Science Festival 2016. We joined scientists from across Oxford to exhibit work from neurology, quantum computers and even the future of our diet (FYI… its insects!); we, of course, brought the microscopes! Our stall was a joint effort with members of the <a href="https://www.dpag.ox.ac.uk/team/shankar-srinivas">Shrinivas Group</a> in the Department of Physiology, Anatomy & Genetics, focusing on developmental biology. The stall was packed with chick and zebrafish embryos and dissected mouse hearts at different stages of their development, as well as live zebrafish embryos under a microscope connected to a screen via a raspberry pi camera from Micron. Children and adults alike gasped wide-eyed at the sight of the beating heart on screen, visible through the transparent body of the zebrafish.
On the opposite end of the stall we set up one of Micron’s Makerbot 3D printers to print mouse hearts at different stages of development. These pieces were added to a whole range of pieces already printed, from embryo sections and developing hearts to GFP (with green lights included!). As further visual aids we projected movies and microscope time-lapse experiments of the development of zebrafish, chick, mouse and human embryos.</p>
<p>Ezequiel Miron from the <a href="http://www.schermellehlab.org/">Schermelleh group</a> at Micron took the data used in making the 3D printed models to generate explorable giant scale replicas within the world of the game Minecraft. These virtual worlds were projected on the big screen and generated a lot of interest among the young boys visiting our stall, and the parents who wanted to know where to download them for their kids to explore at home.</p>
<p>The entire event was a great success, but one moment sticks out as the best. When a couple visited our stall and where handed 3D heart models to inspect, the man looked with interest at the objects, but the woman only caressed it with her fingers, looking distant. Then a beaming smile appeared on her face, she was blind, and so these printed replicates could convey to her what our microscopes, movies and virtual worlds could not.</p>
<p>That achievement along with many other smiling faces made this event a wonderful experience in public engagement.</p>
<p>The Minecraft worlds of the developing heart and the mammalian nucleus can be found at:<a href="http://www.schermellehlab.org/minecraft/">
http://www.schermellehlab.org/minecraft/</a></p>
<p>Many more pictures of this event can be found at:
<a href="http://www.schermellehlab.org/oxsci/">http://www.schermellehlab.org/oxsci/</a></p>

 <img src="images/scifair.png" alt="ScienceFestival" width="566" height="500" align="right" />
 <h4>Micron at the Oxford Science Fair 2016</h4>
 
 <p>&nbsp;</p>
        <p>&nbsp;</p>

        
        <h4>Latest Micron Publication - <a href="http://link.springer.com/protocol/10.1007%2F978-1-4939-3631-1_10">Miron et al., Methods Mol Biol. 2016</a></h4>
        
        <p>The super-resolution three-dimensional structured illumination microscope (3D-SIM) at Micron enables us to study sub-chromosomal chromatin organization in single cells. In this publication we provide methods for pulse-chase replication labeling of individual chromosome territories and replication domain units in mammalian cell nuclei. We describe the application of F-ara-EdU for the in situ detection of segregated chromosome territories with minimized cytotoxicity. Furthermore, we provide detailed protocols for highly efficient electroporation-based delivery or scratch loading of cell impermeable fluorescent nucleotides for live cell studies.</p>
        
        <img src="images/miron.png" alt="ChromatinOrganization_3DSIM" width="566" height="261" align="right" />
 <h4>Chromosome territory segregation imaged with 3D-SIM. (a) Segregation of F-ara-EdU labeled chromosome territories in mouse C127 cells over 6 days after pulse labeling. At day 4 individual territories can be distinguished. Scale bar, 5 μm. (b) Magnified view of consecutive z-sections through the structure of a single chromosome territory (boxed region in a). Scale bar 2 μm.</h4>

         <p>&nbsp;</p>
        <p>&nbsp;</p>



        
        <h4>Micron Publication - <a href="http://www.nature.com/articles/srep27290">Wegel et al., Sci Reports 2016</a></h4>
        
        <p>Micron enables researchers from Oxford (and beyond) to access each of the established super-resolution methods such as Structured Illumination Microscopy (SIM), Stimulated Emission Depletion (STED) microscopy and Single Molecule Localisation Microscopy (SMLM).</p>

<p>Each of these techniques enables an increase in image resolution beyond the classical diffraction-limit, but how do they compare on a practical level? Here, Wegel et al. assess the individual strengths and weaknesses of each technique by imaging a variety of different subcellular structures in fixed cells. We chose examples ranging from well separated vesicles to densely packed three dimensional filaments. We used quantitative and correlative analyses to assess the performance of SIM, STED and SMLM with the aim of establishing a rough guideline regarding the suitability for typical applications and to highlight pitfalls associated with the different techniques.</p>

<img src="images/wegel.png" alt="SIMCheck" width="566" height="205" align="right" />
 <h4>Centrioles in Drosophila primary spermatocytes. Asterless (Asl) localisation in an orthogonal centriole pair was detected with primary and secondary antibodies, the latter coupled to Alexa Fluor 488 for SIM, STED and SMLM. 100% available depletion laser power was used for STED. Scale bar, 0.5 μm. Graph shows FWHM measurements of 60 centriole walls per super-resolution technique.</h4>

         <p>&nbsp;</p>
        <p>&nbsp;</p>
       
        <h4>Micron Publication - <a href="http://science.sciencemag.org/content/351/6277/1094">Uphoff et al., Science 2016</a></h4>
        <p>New study reveals a link between chance events and the formation of mutations</p>
<p>At the molecular level, all biological processes are subject to chance, governed by random encounters of molecules within cells. This means that even cells with identical genetic makeup will show random variation in their behaviour. Microscopy at Micron helped to explore whether such variation might affect the formation of mutations in the genome. The unexpected results have now been published in <a href="http://science.sciencemag.org/content/351/6277/1094">Science</a>.</p>

<p>All organisms rely on proteins that sense and repair the DNA damage that is constantly caused by their own metabolism and by environmental agents such as UV light or mutagenic chemicals. To investigate cell-to-cell variation in DNA repair, Stephan Uphoff and co-workers from Johan Paulsson’s lab at Harvard Medical School decided to revisit a classic DNA damage response that was discovered in the bacterium Escherichia coli back in 1977. At Micron in Oxford, they were able to use modern super-resolution fluorescence microscopy to image individual proteins as they search for and repair DNA damage.</p>
<p>To their surprise, the team found that the protein required to sense DNA damage was present at extremely low quantities in cells before damage. Cells had on average only a single copy of that protein, and a significant fraction of the cells had none at all. ‘How many copies each cell contained was purely due to chance – or bad luck for those cells that had none’, comments Dr Uphoff. It turned out that the cells with no copies did not sense the presence of DNA damage. They were thus unable to activate the response that would allow them to repair the damage.</p>
<p>The team developed a new method to visualize mutations under a microscope. These experiments demonstrated that failure to activate the DNA damage response lead to the formation of mutations. The long-standing dogma in biology that genetic differences cause phenotypic diversity is therefore also true in reverse: The capacity of cells to repair their DNA fluctuates randomly, and this phenotypic variation can lead to genetic changes.</p>

 <img src="images/uphoff1.png" alt="SIMCheck" width="566" height="250" align="right" />
 <h4>Escherichia coli cells activate a DNA damage response (yellow) after treatment with DNA damage. Some cells fail to activate the response and accumulate DNA mismatches – the precursors of mutations. The mismatches were visualized by tracking the movement of a mismatch-binding protein (red tracks). Scale bar: 2 µm.</h4>
        <p>&nbsp;</p>
        <p>&nbsp;</p>

        <h4>Micron Publication - <a href="http://www.nature.com/articles/srep15915">Ball et al., Sci Reports 2015</a></h4>
        <p>Three-dimensional structured illumination microscopy (3D-SIM) makes smart use of interference effects between a patterned excitation and fluorescence emission to generate high-contrast multi-dimensional super-resolution images of fixed and living specimen. Its broad application range and compatibility with standard fluorescence dyes makes 3D-SIM an attractive and accessible method for biologists. However, to generate high-quality images and extract biologically meaningful quantitative data can be a challenging task, particularly to non-specialist users.</p>

<p>We have developed SIMcheck, a suite of powerful plugins for ImageJ/Fiji that enables users to identify and avoid common problems and artifacts with 3D-SIM data, and assess resolution and data quality through objective control parameters. Additionally, SIMcheck provides advanced calibration tools and utilities for common image processing tasks. This open-source software is applicable to all commercial and custom platforms and will promote routine application of super-resolution SIM imaging in cell biology.</p>

<p>Find more information and downloads <a href="http://www.micron.ox.ac.uk/software/SIMcheck.html">here</a> 

 <img src="images/simcheck1.jpg" alt="SIMCheck" width="566" height="250" align="right" />
 <h4>SIMcheck is an open source ImageJ plugin suite for super-resolution stuctured illumination microscopy data quality control.</h4>
        <p>&nbsp;</p>
        <p>&nbsp;</p>
 
        <h4>Micron wins award for next phase in its development <a href="http://www.wellcome.ac.uk/Funding/Biomedical-science/Funded-projects/Awards-made/Strategic-Awards/index.htm"></a></h4>
        <p>Micron has been awarded a renewal of its Strategic Award from the Wellcome Trust to continue its state-of-the-art collaborative activities until at least 2020.</p>

<p>Micron's Director Ilan Davis, together with deputy-Director Jordan Raff at the Dunn School, aim to expand the co-ordinating role played by Micron in this new phase, bringing in additional departments and technologies. This includes <a href="http://www.imm.ox.ac.uk/christian-eggeling">Christian Eggeling</a> from the WIMM, <a href="http://www.cncb.ox.ac.uk/team/group-leaders/booth/">Martin Booth</a> from Engineering, Yvonne Jones and Kay Grunewald from the <a href="http://www.well.ox.ac.uk/research-groups">WTCHG</a>, and <a href="https://groups.physics.ox.ac.uk/genemachines/group/">Achillefs Kapanidis </a>from Physics.</p>

<p>The £4.6million Strategic Award will support the purchase or construction of bespoke microscopes, the development of new tools (analytical and chemical), and increase access and support for users.</p>

<p>The new investment, together with substantial support from the University and existing support from the <a href="http://www2.bioch.ox.ac.uk/nanogroup/nano_home.php">MRC</a> and other funders, will allow Micron to continue to develop as one of the UK's leading bio-imaging facilities.</p>

<p>'Micron is an exemplar of how complex technologies can be offered to researchers,' says Professor Davis. 'There's been a shift in biology to big technologies that cannot be supported by single labs, and the field is evolving so fast that individuals cannot keep up with this.'

'The infrastructure required to make the technologies available widely is substantial. It's not a question of simply pressing a button - for example, the data analysis aspect is complex and needs a deep understanding.'</p>

<p>The award will enable the development of new imaging technologies as well as the continued development of existing microscopy systems. Off-the-shelf microscopes purchased from Zeiss, which provide moderate super-resolution and are very sensitive, will expand Micron's capabilities. Two new systems, the Lattice Lightsheet system and the 4Pi-SMS microscope, will be built.</p>


<p>Janelia Campus Nobel Laureate Eric Betzig recently developed the revolutionary Lattice Lightsheet system (<a href="https://www.sciencemag.org/content/346/6208/1257998.full">Science (2014) Vol. 346</a>). It is ideal for rapid and non-disruptive long-term live cell imaging as it has particularly good light efficiency and therefore causes less damage. Christian Eggeling will lead this work at the WIMM where the microscope will be used to look at the immune system responses live. It will be one of the first instruments of its type in the UK.</p>

<p>The single-molecule high-resolution system 4Pi-SMS is tailored to deep imaging of samples. It was developed by <a href="http://medicine.yale.edu/lab/rothman/">Jim Rothman at Yale</a> and a prototype is currently being built at the Gurdon Institute in Cambridge. In Oxford, Martin Booth in the Department of Engineering has developed the necessary adaptive optics, which allow corrections of aberrations, to image deep. He will spearhead the building of a replica microscope making Micron's instrument the third in the world after Yale and the Gurdon Institute.</p>

<p>The continued development of existing microscopy systems will focus on completion of a <a href="http://www.micron.ox.ac.uk/facilities/DeepSIM">DeepSIM microscope</a>, which has already progressed well from a long-standing collaboration with Professor John Sedat at UCSF. The system lends itself to specimen manipulation, such as microinjection, at the same time as imaging. Like 4Pi-SMS, it uses adaptive optics and there are no commercial systems currently available. Micron researchers have developed user-friendly software that will make the system easy to use for biologists.</p>

<p>Micron will continue its collaboration with Diamond to improve a version of structured illumination microscopy (a mode of super-resolution microscopy) to be used by researchers alongside X-ray microscopy and EM. The lab of Achillefs Kapanidis, Professor of Biological Physics at Oxford, has developed the NanoImager, a compact desktop microscopy for single molecule analyses, in collaboration with David Sherratt in Biochemistry.</p>

<p>Professor Davis comments that the diversity of expertise across Oxford and willingness of individuals to work together has been crucial to Micron's success. 'The landscape of researchers in Oxford is great and there is a high degree of co-operation between them. We are also fortunate to have tremendous institutional support - both financially and in support of the concept.'</p>

<p>The equipment at Micron is available to everyone across Oxford and where appropriate, outside Oxford.</p> 

<p>Article by Jane Itzhaki</p>



        <h4>Micron Publication - <a href="http://www.sciencedirect.com/science/article/pii/S1046202315300190">Demmerle et al., Methods 2015</a></h4>
        <p>At Micron, we frequently receive two critical questions from scientists interested in applying super-resolution technologies. Firstly, what is the actual resolution of the resulting image from a given technique? Secondly, is the increased effort and expense of super-resolution, as compared to conventional microscopy, worth the increase in resolution? These questions are not simple to answer, particularly as there are fundamental differences in super-resolution modalities, which mean that “resolution” is not a directly transferrable metric between techniques. To meaningfully compare different techniques it is essential to specify the exact definition of the resolution one is measuring. Further, resolution is a multifaceted concept and measuring it is not as trivial as it may appear in the literature. For these reasons we felt it necessary to revisit the concept of optical resolution, and developed a method to provide comparable measures of resolution for multiple super-resolution modalities.</p>
        <img src="images/demmerle1.png" alt="Resolution assessment" width="566" height="372" align="right" />
        <p>We introduce the technique of Fourier Spectrum Analysis for comparing resolution in super-resolution microscopy. We generated comparable images of microtubules in all three super-resolution modalities: 3D-SIM on our OMX V3 instrument; gSTED on the Wolfson Imaging Centre’s Leica SP8 instrument, and dSTORM (SMLM) on our OMX V2 system. Using Fourier Transform and Radial Profile tools from ImageJ, we can objectively compare the entirety of these images in Fourier Space to determine resolution, avoiding many of the issues with the traditional resolution measurement of calculating the full-width half maximum of fitted Gaussian profiles over particular selected objects.</p>
        <p>Using this method confirms some traditional resolution measurements, but also raises questions about the way we conceptualize and quantify both resolution itself and the differences between super-resolution modalities. These fundamental contrasts make cross-platform comparisons difficult, and inform the specific biological conclusions the researcher can draw from an experiment. Further, they emphasize the importance of developing each imaging workflow with consideration of all variables, including spatial resolution but also temporal resolution, spatial correlation, and energy load. In conclusion, a ‘‘resolution number’’ should not be the primary factor in deciding which technique to use.</p>
        <img src="images/demmerle2.png" alt="Resolution assessment" width="566" height="351" align="right" /> 
        <h4>Fourier Space Analysis to compare resolutions</h4>
        <p>&nbsp;</p>
        <p>&nbsp;</p>
        
		<h4>Micron Publication - <a href="http://www.nature.com/srep/2015/150330/srep09583/full/srep09583.html">Johnson et al., Sci Reports 2015</a></h4>
        <p>Correlative light and electron microscopy (CLEM) combines the very distinct and complementary strengths of fluorescence microscopy and electron microscopy. Besides the possibility of imaging living cells, in fluorescence microscopy structures or proteins of interest inside the cell can be labelled very specifically. This can be problematic or often even impossible with electron microscopy. On the other hand, electron microscopy allows structural details to be studied within the cellular context at a resolution in the range of a few nanometres, whereas the resolution in fluorescence microscopy is limited by the diffraction of light to around 200 nm.</p>
        <p>In the past 20 years, so-called super-resolution methods have been developed to overcome this resolution limit in fluorescence microscopy, which was last year recognised with the award of the Nobel price. Super-resolution imaging is helping to bridge the gap between fluorescence and electron microscopy. However, the requirements regarding sample preparation for both imaging modalities are very different. So far, performing super-resolution CLEM for imaging the same cell and combining the super-resolution fluorescence information with the ultrastructure of electron microscopy meant a big compromise. Either the resolution in the fluorescence images was not high enough for clear interpretation of the data or the structural preservation of the sample was deteriorated significantly, for example due to chemical fixation of the sample.</p>
        <img src="images/Johnson1.png" alt="Cryo Super-Resolution" width="566" height="134" align="right" />
         <h4>Comparison of resolution achieved with in-resin super-resolution CLEM. Left: Conventional wide-field fluorescence microscopy. Middle: Super-resolution SMLM. Right: TEM. The structural resolution of ~50 nm achieved with super-resolution SMLM in resin sections using standard fluorescence proteins (here mVenus) allowed a superior correlation of fluorescent signals and EM ultrastructure.</h4>
        <p>We have developed a method that ensures superior structural preservation of the sample whilst allowing uncompromised super-resolution imaging and sufficient contrast for high quality electron microscopy. In the first step, the sample is high pressure frozen, which immobilises it in glass-like amorphous ice (vitrification) and preserves the structure in a near-native state. A process of freeze substitution and resin embedding follows, during which the ice is slowly replaced by acetone, and then the acetone is replaced with resin, after which it is polymerised using UV light at a low temperature. The key for successful super-resolution CLEM using this approach is to preserve throughout the whole process the fluorescence in the sample and the photo-switching ability of the fluorescent molecules, on which super-resolution microscopy is crucially dependent on. We have identified the critical parameters during the process of high pressure freezing, freeze substitution and resin embedding and developed a method which enables in-resin super-resolution microscopy using standard fluorescent proteins at a resolution better than 50 nm followed by electron microscopy imaging of ultrathin sections. We found that the addition of tannic acid to the freeze substitution medium has a huge effect on the photo-switching of the fluorescent molecules, but at a certain level it also negatively affects the structural preservation. Here, it was important to find the right concentration of tannic acid to achieve optimal results with both imaging modalities. We demonstrate that our method works for a variety of different standard fluorescent proteins such as GFP, mVenus (an improved YFP) and mRuby2 (an improved RFP). Not having to use special photo-switchable fluorophores allows a quick and broad application of our method in the field of cell biology.</p>
         <img src="images/Johnson2.png" alt="Cryo Super-Resolution" width="566" height="447" align="right" />
         <h4>Correlative in-resin super-resolution fluorescence and EM imaging of HEK293T cells transfected with EphA2/A4 receptor proteins fused to mGFP, mVenus or mRuby. The first column shows conventional wide-field fluorescence images, the second column the corresponding super-resolution SMLM (single molecule localization microscopy) images. TEM (transmission electron microscopy) images of the same cells are depicted in the third column (plasma membrane (PM), endoplasmic reticulum (ER) and the nucleus (NUC) are indicated in the individual images), followed by an overlay of the TEM images with the conventional wide-field fluorescence images, and an overlay of TEM and super-resolution images.</h4>
         <p>&nbsp;</p>
        <p>&nbsp;</p>
        <h4>Micron Publication - <a href="http://elifesciences.org/content/3/e03399">Conduit et al., eLife 2014.</a></h4>
        <img src="images/conduit1.png" alt="Conduit Live two colour" width="350" align="right" />

        <p>
          Centrosomes are cellular organelles composed of two centrioles
          surrounded by a matrix of pericentriolar material (PCM). The PCM
          expands dramatically in size in mitosis, but it is unclear how this
          process occurs. Using a new super-resolution imaging technique
          developed at Micron, Raff and colleagues now clarify how the PCM
          matures.
        </p>
        <p>
          PCM maturation is an important step in mitosis. The increase in PCM
          size is correlated with an increase in microtubule nucleation ability,
          important for the formation of the mitotic spindle and correct
          chromosome segregation. The PCM is formed by hundreds of different
          proteins, but work now published by the Raff lab (Dunn School) in
          eLife suggests that this process relies on the interplay between
          only a few proteins.
        </p>
        <p>
          Raff and colleagues had previously used a technique known as
          "Fluorescence Recovery After Photobleaching" (FRAP) to show that
          the PCM protein Cnn displayed an unusual dynamic behavior, whereby
          it was only incorporated into the central region of the PCM and then
          spread outwards. Modifying the amount of Cnn in the PCM caused a
          corresponding change in the amount of PCM, suggesting that Cnn may
          form a scaffold that spreads away from the centrioles to support the
          extended PCM structure. In this paper, Raff and colleagues used FRAP
          to show that only one other protein, Spd-2, displayed a similar
          behaviour to Cnn and could also support PCM assembly. It was
          unclear, however, how these two scaffolds related to each other,
          particularly as standard resolution microscopy could not easily
          distinguish between them.
        </p>
        <p>
          The authors therefore turned to super-resolution structured
          illumination (3D-SIM) microscopy, available at Micron. Live two
          colour 3D-SIM revealed that the Spd2 and Cnn scaffolds only
          partially overlapped. This suggested that the two proteins behave
          differently, but in what way? To address this question, Micron
          developed a new technique that combined 3D-SIM with FRAP. This
          allowed the authors to examine the dynamics of Cnn and Spd-2 in
          unprecedented detail. Remarkably, this showed a clear difference
          between the two scaffolds: Spd-2 is initially recruited around the
          wall of the centriole and then moves outwards, whereas Cnn is
          initially recruited into a broader area and then spreads further
          outwards.
        </p>
        <p>
          These observations led the authors to carry out further experiments
          that collectively suggested a simple model for PCM maturation.
          Spd-2 is recruited to the centriole wall and then helps recruit Cnn.
          Cnn is modified by the kinase Polo/Plk1 and builds a scaffold that
          supports the outward movement of Spd-2. As Spd-2 moves outwards, it
          creates a larger area for Cnn recruitment, which in turn provides
          more support for the outward movement of Spd-2. Thus, a positive
          feedback loop is formed that allows the rapid expansion of the two
          scaffolds, which recruit the rest of the PCM, during mitosis.
        </p>

        <img src="images/conduit2.png" alt="conduit 3D-SIM FRAP" width="566" align="right" />
        <h4>3D-SIM FRAP on (A) DSpd2-GFP and (B) GFP-Cnn </h4>
        <p>&nbsp;</p>
        <p>&nbsp;</p>
        

        <h4>Micron Publication - <a href="http://pubs.acs.org/doi/full/10.1021/nl501870p">Kaufmann et al., Nano Letters. 2014.</a></h4>
        <img src="images/cryo2.png" alt="Rainer Graphical Abstract" width="350" align="right" />

        <p>
          Studying biological structures with fine details does not only
          require a microscope with high resolution, but also a sample
          preparation process that preserves the structures in a near-native
          state. Live-cell imaging is restricted mostly to the field of light
          microscopy. For studies requiring much higher resolution, fast
          freezing techniques (vitrification) are successfully used to
          immobilize the sample in a near-native state for imaging with
          electron and X-ray cryo-microscopy. Fluorescence cryo-microscopy
          combines imaging of vitrified samples with the advantages of
          fluorescence labeling of biological structures, but the advantages
          of vitrified specimens have not been fully exploited to date in
          fluorescence microscopy, because of the limited resolution of
          400-500 nm due to the lack of cryo-immersion objectives.
        </p>
        <p>
          We demonstrated how super-resolution imaging can be successfully
          applied for fluorescence cryo-microscopy to increase the resolution
          up to a factor of 5 at a temperature of -192°C. Using standard
          fluorescent proteins we were able to image biological structures
          in vitrified cells with a resolution of 125 nm and could locate
          individual molecules with an average precision of 40 nm. This
          technique is highly complementary to electron and X-ray
          cryo-microscopy and presents a powerful tool for imaging cellular
          structures in a near-native state at nanometer resolution.
        </p>

        <img src="images/cryo3.png" alt="Cryo Super-Resolution" width="566" height="404" align="right" />

        <h4>
          Single molecule super-resolution cryo-imaging.
          (A) Wide-field fluorescence cryo-microscopy image of endoplasmic
          reticulum labeled with mVenus in a vitrified cell. Resolution: ~450 nm.
          (B) Corresponding cryo super-resolution image of single molecule
          localization cryo-microscopy. Average localization accuracy: 42 nm.
          Structural resolution: ~125 nm. Color coding indicates local densities
          of detected single molecule positions as well as the corresponding
          Nyquist resolution.
        </h4>

        </br>

        <h4>BSCB Competition Winner 2012</h4>
        <p>
          Congratulations to Sheng-Wen Chui from the Department of
          Biochemistry at the University of Oxford. Sheng-Wen used Micron's
          <a href="facilities/zippy">Delta-Vision microscope</a> to
          take this superb image of filamentous cells of the bacterium
          Rhodobacter sphaeroides.
        </p>
        <img src="images/SW_Chiu.jpg" width="560" align="right" />
        <h4>
          The Tubulin homolog FtsZ (tagged with CFP) forms dot-like and spiral
          structures in two distinct populations. The FtsZ cytoskeleton affects
          the localization of the membrane chemosensory protein clusters (YFP).
          Cell bodies are shown in magenta.
        </h4>

        </br>

        <h4>Wytham Woods Exhibition</h4>
        <p>
          Out of the Woods is an exhibition on Wytham Woods north of Oxford,
          the best studied piece of wood land in the world. The exhibition
          features prints and wood sculpture as well as photographs about
          life in the woods. We have contributed three large digital images
          that use fluorescence and DIC to capture the beauty of cells and
          organelles in bluebell leaves from Wytham Woods. The exhibition
          runs until 30 September 2012.
        </p>
        <img src="images/2_stomata.jpg"
          alt="image" width="560"align="right" />
        <h4>
          Epidermis (outer layer) of a bluebell leaf showing the guard cells
          of two stomata and red chlorophyll autofluorescence in the
          chloroplasts contained within the guard cells.
        </h4>
        <p>
          Stomata are closable pores through which gas is exchanged between
          the leaf and the environment. During the day they allow oxygen and
          water vapour to leave the leaf and air to enter while during the
          night carbon dioxide is expelled. The guard cells are the only
          cells in the epidermis that contain chloroplasts where
          photosynthesis takes place. The tissue shown in this image is in
          reality 100&#956;m x 100&#956;m (=0.1 mm x 0.1 mm) in size.
        </p>
        <img src="images/parenchyma_cell.jpg" width="560" align="right"/>
        <h4>Parenchyma (inner layer) cell of a bluebell leaf.</h4>

        <p>
          Most of the inner cells of leaves contain chloroplasts (chl) in
          the thin rim of cytoplasm around the edge of the cell. This rim
          also contains a nucleus (n, with chloroplasts in front and behind)
          and many other much smaller organelles (specialised compartments).
          The vast majority of the plant cell is made up of a vacuole (v)
          filled with water and soluble substances. The cell section in this
          image is in reality 75&#956;m (=0.075 mm) wide.
        </p>

        <img src="images/chlorophyll_autofluorescence.jpg" width="560"
          align="right"/>
        <h4>Chlorophyll autofluorescence in chloroplasts of a parenchyma cell in a bluebell leaf.</h4>

        <p>
          Chloroplasts are the compartments in a plant cell in which
          photosynthesis takes place. Photosynthesis is a process that uses
          light, water and carbon dioxide to produce sugars and oxygen.
          Without photosynthesis the earth would only be populated by
          bacteria. The network of membranes inside chloroplasts contains
          chlorophyll, which absorbs mostly blue and red light, and emits
          light in the far red part of the spectrum. The absorbed light is
          used for photosynthesis. The emitted fluorescent light from several
          chloroplasts is shown in this image. The biggest of these
          chloroplasts is 14&#956;m (=0.014 mm) in length.
        </p>

        <h3>External news &amp; events</h3>

        <ul>
          <li><a href="http://wellcometrust.wordpress.com/">the Wellcome Trust blog</a></li>
          <li><a href="http://www.oxfordsparks.net">Oxford Sparks</a> - Website and blog describing cutting edge science going on around the University.</li>
        </ul>

      </div>

      <!-- News and events from twitter  -->
      <a class="twitter-timeline" width="290" height="3000"
        data-chrome="nofooter noheader" data-border-color="#4e6983"
        href="https://twitter.com/MicronOxford" data-widget-id="293683357219422209">Tweets by @MicronOxford</a>
      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>

    </div>
    <?php include "footer.html"; ?>
  </div>
</body>

</html>
