<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">

<html lang="en-GB">

<head>
  <title>Micron Oxford Advanced Bioimaging: research page</title>
  <meta name=description content="Micron Oxford Advanced Bioimaging Research">
  <meta name=keywords content="advanced bioimaging microscopy analysis processing super-resolution localization microscopy SPIM TIRF">
  <meta http-equiv="Content-Type" CONTENT="text/html; charset=UTF-8">
  <link rel="stylesheet" type="text/css" href="/styles/main.css" >
  <link rel="stylesheet" type="text/css" href="/styles/research.css" >
  <link rel="icon" type="image/ico" href="/images/MicronMu.ico" >

</head>

<body id="research">
  <div id="container_all">
    <?php include "../headnav.html"; ?>
    <div id="mainbody">

      <div id="leftpanel">

        <h3>Micron research aims to:</h3>
        <ol id="research_aims">
          <li>To create a unique community and infrastructure allowing
          outstanding research groups to address key questions in their
          fields using advanced imaging tools such as super-resolution
          microscopy and single molecule imaging in living cells. Our
          overarching goal is to use light microscopy to study "vivo
          biochemistry". In other words to observe and interfere with
          biomolecules in the context of living cells.</li>

          <li>To develop and apply new bioimaging technologies. Micron
          associated groups have been using advanced microscopy methods to
          tackle major biological problems in a number of fields in cell
          biology. These include: chromosome and RNA biology, development,
          epigenetics, stem cells, non-coding RNAs, intracellular trafficking
          and signalling.</li>

          <li>To forge links and collaborations with a variety of other
          overlapping disciplines that have an outstanding presence in
          Oxford, including: systems biology, physics, engineering and
          computer sciences.</li>
        </ol>

        <div>
          <h3>Advanced Bioimaging</h3>
          <p>
            Research in micron involves technology development in all aspects
            of microscopy: specimen preparation, probe development, instrument
            development and image analysis. However, we are focusing on the
            following approaches:
          </p>
          <ul>
            <li>Super-resolution imaging using <a href="/research/super_resolution_microscopy">structured illumination</a></li>
            <li>Super-resolution imaging and single molecule dynamics using <a href="/research/localization-microscopy">localization microscopy</a></li>
            <li><a href="/facilities/lightsheet">Light sheet microscopy</a> and tomography</li>
            <li>Advanced Total Internal Reflection Microscopy methods</li>
            <li>Advanced multidimensional live cell imaging using the <a href="/facilities/OMXV3">OMX microscope</a></li>
          </ul>
        </div>

        <div>
          <h3>Image Analysis</h3>
          <p>
            We have two major goals in terms of image analysis in Micron: firstly, to
            encourage and help researchers to adopt the best available image analysis 
            tools and workflow; and secondly, to develop transformational new imaging 
            tools and apply existing cutting edge methodologies.
          </p>
          <p>Current activities include:</p>
          <ul>
            <li>Setting up computer infrastructure and software</li>
            <li>Developing an efficient workflow for super-resolution data</li>
            <li>Preparing image analysis lecture and course material</li>
            <li>Development of <a href="/software/SIMCheck">SIMCheck</a> our our Image J plug-in that allows users to apply quality control to their Structured illumination super-resolution systems.</li>
            <li>Developing an improved particle tracker and adding functionality to
                <a href="http://emonet.biology.yale.edu/microbetracker/">MicrobeTracker</a></li>
                <li>Development of <a href="http://www.darogan.co.uk/ParticleStats/">'ParticleStats'</a> software for the analysis of analysis of intracellular particle motility and cytoskeletal polarity </li>
                
          </ul>
        </div>
        <div>
        <h3>Developmental microscope systems</h3>
        <ul>
        <li><a href="/facilities/DeepSIM">DeepSIM</a> - upright 3D-SIM microscope that uses adaptive optics for imaging deep in living specimens while allowing easy manipulation (microinjection and electrophysiology).</li>
        <li><a href="/facilities/OMX-T">OMX-T</a> Cryo SIM/STORM system which enables super-resolution imaging of frozen samples, providing significant gains in specimen preservation and photo-stability.</li>
        <li>4Pi-SMS single molecule interferometric microscope - dependent on funding</li>
        <li>Lattice Light Sheet Microscope - dependent on funding</li>
        
        </ul>
        

</div>
      </div>

      <div id="rightpanel">
        <h3>Publication Highlights</h3>
        click <a href="/research/publications">here</a> for all publications
        <h4>2018</h4>
        <ul class="publicationlist">
          <li>
            Titlow, Joshua S., Lu Yang, Richard M. Parton, Ana
            Palanca, and Ilan Davis. "Super-Resolution Single Molecule
            FISH at the Drosophila Neuromuscular Junction." In RNA
            Detection, pp. 163-175. Humana Press, New York, NY, 2018.
          </li>
          
        </ul>

        <h4>2017</h4>
        <ul class="publicationlist">
        <li>Diaconeasa Z, Ayvaz H, Ruginǎ D, Leopold L, Stǎnilǎ A, Socaciu C, Tăbăran F, Luput L, Mada DC, Pintea, Jefferson. Melanoma Inhibition by Anthocyanins Is Associated with the Reduction of Oxidative Stress Biomarkers and Changes in Mitochondrial Membrane Potential. Plant Foods Hum Nutr. 2017 Dec;72(4):404-410.</li>
          
          <li>
            Gartenmann L, Wainman A, Qurashi M, Kaufmann R, Schubert S, Raff JW, Dobbie IM. A combined 3D-SIM/SMLM approach allows centriole proteins to be localized with a precision of ∼4-5 nm. Curr Biol. 2017 Oct 9;27(19):R1054-R1055.
          </li>
          
          <li>
            Almeida M, Pintacuda G, Masui O, Koseki Y, Gdula M, Cerase A, Brown D, Mould A, Innocent C, Nakayama M, Schermelleh L, Nesterova TB, Koseki H, Brockdorff N. PCGF3/5-PRC1 initiates Polycomb recruitment in X chromosome inactivation. Science. 2017 Jun 9;356(6342):1081-1084. 
          </li>
          
          <li>
          Feng Z, Caballe A, Wainman A, Johnson S, Haensele AFM, Cottee MA, Conduit PT, Lea SM, Raff JW. Structural Basis for Mitotic Centrosome Assembly in Flies. Cell. 2017 Jun 1;169(6):1078-1089.e13. 
          </li>
          
          <li>
            Demmerle J, Innocent C, North AJ, Ball G, Müller M, Miron E, Matsuda A, Dobbie IM, Markaki Y, Schermelleh L.  Strategic and practical guidelines for successful structured illumination microscopy. Nat Protoc. 2017 May;12(5):988-1010.
          </li>
          
          <li>
            Kraus F, Miron E, Demmerle J, Chitiashvili T, Budco A, Alle Q, Matsuda A, Leonhardt H, Schermelleh L, Markaki Y. Quantitative 3D structured illumination microscopy of nuclear structures. Nat Protoc. 2017 May;12(5):1011-1028.
          </li>
          
        </ul>
        
          <h4>2016</h4>
        <ul class="publicationlist">
        <li>Stephan Uphoff1, Nathan D. Lord, Burak Okumus, Laurent Potvin-Trottier, David J. Sherratt, Johan Paulsson. Stochastic activation of a DNA damage response causes cell-to-cell mutation rate variation. Science. 2016 Mar 4;351(6277):1094-7.</li>
        <li>Nolivos S, Upton AL, Badrinarayanan A, Müller J, Zawadzka K, Wiktor J, Gill A, Arciszewska L, Nicolas E, Sherratt D. MatP regulates the coordinated action of topoisomerase IV and MukBEF in chromosome segregation. Nat Commun. 2016 Jan 28;7:10466.</li>
          
        </ul>
        
        <h4>2015</h4>
        <ul class="publicationlist">
        <li>Hagen C, Dent KC, Zeev-Ben-Mordehai T, Grange M, Bosse JB, Whittle C, Klupp BG, Siebert CA, Vasishtan D, Bäuerlein FJ, Cheleski J, Werner S, Guttmann P, Rehbein S, Henzler K, Demmerle J, Adler B, Koszinowski U, Schermelleh L, Schneider G, Enquist LW, Plitzko JM, Mettenleiter TC, Grünewald K. Structural Basis of Vesicle Formation at the Inner Nuclear Membrane. Cell. 2015 Dec 17;163(7):1692-701. </li>
           <li>Ball G, Demmerle J, Kaufmann R, Davis I, Dobbie IM, Schermelleh L. SIMcheck: a Toolbox for Successful Super-resolution Structured Illumination Microscopy. Sci Rep. 2015 Nov 3;5:15915.</li>
        
        <li>Demmerle J, Wegel E, Schermelleh L, Dobbie IM. Assessing resolution in super-resolution imaging. Methods. (2015) Jul 8. pii: S1046-2023(15)30019-0.</li>
        <li>Houlard M, Godwin J, Metson J, Lee J, Hirano T, Nasmyth K. (2015). Condensin confers the longitudinal rigidity of chromosomes. Nat Cell Biol. 2015 May 11. </li>
          <li>Errin Johnson, Elena Seiradake, E. Yvonne Jones, Ilan Davis, Kay Grunewald & Rainer Kaufmann (2015). 
          Correlative in-resin super-resolution and electron microscopy using standard
fluorescent proteins. Scientific Reports 5; 9583.</li>
</ul>
        <h4>2014</h4>
        <ul class="publicationlist">
          <li>
            Lesterlin C, Ball G, Schermelleh L, Sherratt D. 2014. DRecA bundles
            mediate homology pairing between distant sisters during DNA break
            repair. Nature. 506: 249-53.
          </li>
          <li>
            Fleurie A, Lesterlin C, Manuse S, Zhao C, Cluzel C, Lavergne JP,
            Franz-Wachtel M, Macek B, Combet C, Kuru E, Van Nieuwenhze MS,
            Brun YV, Sherratt D, Grangeasse C. MapZ marks the division sites
            and positions FtsZ rings in Streptococcus pneumoniae.
            Nature. 2014 Nov 26
          </li>
          <li>
            Conduit PT, Richens JH, Wainman A, Holder J, Vicente CC, Pratt MB,
            Dix CI, Novak ZA, Dobbie IM, Schermelleh L, Raff JW. A molecular
            mechanism of mitotic centrosome assembly in Drosophila. Elife.
            2014 Aug 22;3
          </li>
          <li>
            R. Kaufmann, P. Schellenberger, E. Seiradake, I. Dobbie, E. Y.
            Jones, I. Davis, C. Hagen, K. Grünewald (2014) Super-Resolution
            Microscopy using Standard Fluorescent Proteins in Intact Cells
            under Cryo-Conditions. Nano Letters, 14 (7), 4171-4175.
          </li>
          <li>
            Paul T. Conduit, Zhe Feng, Jennifer H. Richens, Janina Baumbach,
            Alan Wainman, Suruchi D. Bakshi, Jeroen Dobbelaere, Steven Johnson,
            Susan M. Lea, and Jordan W. Raff.  The Centrosome-Specific
            Phosphorylation of Cnn by Polo/Plk1 Drives Cnn Scaffold Assembly
            and Centrosome Maturation. Developmental Cell, 20 March 2014
          </li>
          <li>
            Cerase A, Smeets D, Tang AY, Gdula M, Kraus F, Spivakov M,
            Moindrot B, Leleu M, Tattermusch A, Demmerle J, Nesterova T,
            Green C, Otte AP, Schermelleh L, Brockdorff N. 2014. Spatial
            separation of Xist RNA and polycomb proteins revealed by
            super-resolution microscopy. Proc Natl Acad Sci USA 111: 2235-40.
          </li>
        </ul>

        <h4>2013</h4>
        <ul class="publicationlist">
          <li>
            Lesterlin C, Ball G, Schermelleh L, Sherratt DJ. RecA bundles
            mediate homology pairing between distant sisters during DNA break
            repair. Nature. 2013 Dec 22.
          </li>
          <li>
            Hahn M, Dambacher S, Dulev S, Kuznetsova AY, Eck S, Wörz S, Sadic D,
            Schulte M, Mallm JP, Maiser A, Debs P, von Melchner H, Leonhardt H,
            Schermelleh L, Rohr K, Rippe K, Storchova Z, Schotta G. Suv4-20h2
            mediates chromatin compaction and is important for cohesin
            recruitment to heterochromatin. Genes Dev. 2013 Apr 15;27(8):859-72.
          </li>
          <li>
            Uphoff S, Reyes-Lamothe R, Garza de Leon F, Sherratt DJ,
            Kapanidis AN. Single-molecule DNA repair in live bacteria.
            Proc Natl Acad Sci U S A. 2013 May 14;110(20):8063-8.
          </li>
        </ul>

        <h4>2012</h4>
        <ul class="publicationlist">
          <li>
            Badrinarayanan A, Reyes-Lamothe R, Uphoff S, Leake MC, Sherratt DJ.
            In vivo architecture and action of bacterial structural maintenance
            of chromosome proteins. Science. 2012 Oct 26;338(6106):528-31.
          </li>
          <li>
            Brown AC, Dobbie IM, Alakoskela JM, Davis I, Davis DM.
            Super-resolution imaging of remodelled synaptic actin reveals
            different synergies between NK cell receptors and integrins.
            Blood. 2012 Sep 10.
          </li>
          <li>
            Roque H, Wainman A, Richens J, Kozyrska K, Franz A, Raff JW.
            Drosophila Cep135/Bld10 maintains proper centriole structure but is
            dispensable for cartwheel formation. J Cell Sci. 2012 Sep 12.
          </li>
          <li>
            Dunsch AK, Hammond D, Lloyd J, Schermelleh L, Gruneberg U, Barr FA.
            Dynein light chain 1 and a spindle-associated adaptor promote
            dynein asymmetry and spindle orientation.
            J Cell Biol. 2012 Sep 17;198(6):1039-54. Epub 2012 Sep 10.
          </li>
        </ul>

       
        </ul>
      </div>

    </div>
    <?php include "../footer.html"; ?>
  </div>
</body>

</html>
